src/main/java/hudson/plugins/jacoco/JacocoBuildAction.java

```diff
@@ -359,4 +355,12 @@ public final PrintStream getLogger() {
 	    // does not run the construct and thus leaves the transient variables empty
 	    return System.out;
 	}
+
+	public void onAttached(Run<?, ?> run) {
```

Comment:
the version of Jenkins (1.609.3) you are now targeting uses 1.6 - so you can fix that :-)